### PR TITLE
update flix with examples and link backs

### PR DESCRIPTION
**Pull request by bthaile**
closes: https://github.com/onflow/docs/issues/502

Update FLIX documentation for flow-cli and advanced topics to link to each other. 
Add clarification on how to use FLIX across different teams.

### CODE REVIEW

File: docs/build/advanced-concepts/flix.md [hunk 1]
```diff
@@ -8,18 +8,49 @@ Flow Interaction Templates is a standard for how contract developers, wallets, u
...
+ - https://github.com/onflow/hello-world-flix/blob/v0.1.0/cadence/templates/ReadHelloWorld.template.json
+ - https://github.com/onflow/hello-world-flix/blob/v0.1.0/cadence/templates/UpdateHelloWorld.template.json
+
+Developers can use FLIX templates from the smart contract github to interact with their smart contracts. They simply need the FLIX template URLs to create binding files (TypeScript or JavaScript). One major benefit is the web developers don't need to learn Cadence or copy Cadence to their repository in order to integrate with existing smart contracts. 
+
+TypeScript code generated from templates:
+- https://github.com/onflow/hello-world-web/blob/main/app/cadence/readHelloWorld.ts
+- https://github.com/onflow/hello-world-web/blob/main/app/cadence/updateHelloWorld.ts
+
+::: warning
```
Comment (nialexsan) [inline docs/build/advanced-concepts/flix.md:45]:
```suggestion
:::warning
```

File: docs/build/advanced-concepts/flix.md [hunk 2]
```diff
@@ -8,18 +8,49 @@ Flow Interaction Templates is a standard for how contract developers, wallets, u
...
+
+manually added @ts-ignore, fixes to fcl types will correct these linting issue
+
+:::
+
+See the `hello-world-web` [README](https://github.com/onflow/hello-world-web/tree/main) for more information on how to generate and execute FLIX templates here [flow-cli flix](../../tools/flow-cli/flix) commands
```
Comment (nialexsan) [inline docs/build/advanced-concepts/flix.md:51]:
```suggestion
See the `hello-world-web` [README](https://github.com/onflow/hello-world-web/tree/main) for more information on how to generate and execute FLIX templates here [flow-cli flix](../../tools/flow-cli/flix.md) commands 
```

File: docs/build/advanced-concepts/flix.md [hunk 3]
```diff
@@ -8,18 +8,49 @@ Flow Interaction Templates is a standard for how contract developers, wallets, u
...
+
+Developers can use FLIX templates from the smart contract github to interact with their smart contracts. They simply need the FLIX template URLs to create binding files (TypeScript or JavaScript). One major benefit is the web developers don't need to learn Cadence or copy Cadence to their repository in order to integrate with existing smart contracts. 
+
+TypeScript code generated from templates:
+- https://github.com/onflow/hello-world-web/blob/main/app/cadence/readHelloWorld.ts
+- https://github.com/onflow/hello-world-web/blob/main/app/cadence/updateHelloWorld.ts
+
+::: warning
+
+manually added @ts-ignore, fixes to fcl types will correct these linting issue
```
Comment (nialexsan) [inline docs/build/advanced-concepts/flix.md:47]: could you rephrase this sentence? I'm not sure what it means

File: docs/build/advanced-concepts/flix.md [hunk 4]
```diff
@@ -8,18 +8,49 @@ Flow Interaction Templates is a standard for how contract developers, wallets, u
...
 
-By using FLIX transactions and scripts, develoeprs don’t have to write their own for common operations!
+By using FLIX transactions and scripts, developers don’t have to write their own for common operations!
 
 Read more about the design and purpose of FLIX in the [FLIP](https://github.com/onflow/flips/blob/main/application/20220503-interaction-templates.md)
 
 ## Using FLIX
 
-Flow makes FLIX available through an API available at flix.flow.com. Other community run APIs are available, such as flix.ecdao.org. 
+Flow makes FLIX available through an API available at flix.flow.com.
```
Comment (nialexsan) [inline docs/build/advanced-concepts/flix.md:17]: should `flix.flow.com` be a link?

Comment (bthaile) [reply docs/build/advanced-concepts/flix.md:17]: nope, since it's a api and needs full url, which is lower in the markdown

File: docs/build/advanced-concepts/flix.md (hunk 3 above)
Comment (bthaile) [reply docs/build/advanced-concepts/flix.md]: updated PR, thanks

**Review by nialexsan — APPROVED** [review]
